src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -72,6 +72,25 @@ public void ProcessStart_UseShellExecute_OnLinux_ThrowsIfNoProgramInstalled()
             }
         }
 
+        [Fact]
+        [OuterLoop("Opens program")]
+        [PlatformSpecific(TestPlatforms.Linux)]
+        public void ProcessStart_DirectoryNameInCurDirectorySameAsFileNameInExecDirectory_Success()
+        {
+            string fileToOpen = "dotnet";
+            var dotnetFolderInCurDirectory = Path.Combine(Environment.CurrentDirectory, "dotnet");
```

Comment:
Please don't have tests write anything to current directory (as a matter of tidiness). Write to temp. You can change current directory to temp for the duration of the test, if necessary.

Reply:
but the use case is exercising scenario where we have a folder in current directory with same name of the filename we want to run in execution directory.

Reply:
You can change what the current directory is during the test -- the folder it happens to be initially (presumably the test dll directory) is not special. Just change it to temp?